src/commons/utils/alerts.py

```diff
@@ -60,104 +62,176 @@ def send_mail(self, message):
+        :param status_code: Color code used as per execution status.
         :return: Formatted MIME message
         """
-        hctl_status = json.dumps(self.health_obj.get_hctl_status()[1], indent=4)
-        result, pod_status = self.health_obj.execute_command(commands.CMD_POD_STATUS)
-        status = f"Corio TestPlan {str(self.tp_id or '')} is {execution_status} " \
-                 f"on {self.health_obj.host}"
-        subject = status
-        body = f"<h3>{status}.</h2>\n" \
-               f"<h3>PFA hctl cluster status, pod status & execution status.</h3>\n"
-        build_url = os.getenv("BUILD_URL")
-        if build_url:
-            body += f"""Visit Jenkins Job: <a href="{build_url}">{build_url}</a>"""
+        # Mail common parameters.
+        execution_duration = convert_datetime_delta(datetime.now() - self.start_time)
         message = MIMEMultipart()
         message['From'] = self.sender
         message['To'] = COMMASPACE.join(self.receiver.split(','))
         message['Date'] = formatdate(localtime=True)
-        message['Subject'] = subject
+        # Message ID.
         if not self.message_id:
             self.message_id = make_msgid()
             message["Message-ID"] = self.message_id
         else:
             message["In-Reply-To"] = self.message_id
             message["References"] = self.message_id
-        attachment = MIMEApplication(hctl_status, Name="hctl_status.txt")
-        attachment['Content-Disposition'] = 'attachment; filename=hctl_status.txt'
-        message.attach(attachment)
-        if result:
-            attachment = MIMEApplication(pod_status, Name="pod_status.txt")
-            attachment['Content-Disposition'] = 'attachment; filename=pod_status.txt'
+        # Mail subject.
+        message['Subject'] = f"Corio: TestPlan {str(self.tp_id or '')}, Server {self.host}"
+        # Execution status
+        body = "<h2 style='text-align:center;'>Corio Execution Status</h2>"
+        body += f"""<table style='width: 100%; border: thin black dotted; text-align: left;'
+         width = 'nowrap;'> <tr style='background-color:{status_code}'>"""
+        body += f"""<td style='color: white; font-size: 124%; padding-left: 5px; font-weight: bold;'
+         colspan=2><b>Execution status:</b> {execution_status}</td></tr>"""
+        body += f"<tr><td><b>Execution started:</b></td> <td>{self.start_time}</td></tr>"
+        body += f"<tr><td><b>Execution duration:</b></td> <td>{execution_duration}</td></tr>"
+        # Cluster health and pod status.
+        if self.health_check:
+            hctl_status = self.health_obj.get_hctl_status()[1]
+            hctl_data = json.dumps(hctl_status, indent=4)
+            result, pod_status = self.health_obj.execute_command(commands.CMD_POD_STATUS)
+            health_status = "Cluster is healthy"
+            if "offline" in str(hctl_status) or "unknown" in str(hctl_status):
+                health_status = "Cluster is unhealthy"
```

Comment:
Better to keep healthy or non-healthy rule at one place in `check_cluster_health` function of `src/commons/utils/cluster_utils.py` instead of defining those rules here.